deploy/template.yaml

```diff
@@ -58,20 +58,30 @@ Mappings:
       apiTracingEnabled: false
   EnvironmentVariables: # This is all the environment specific environment variables that don't belong in globals.
     dev:
-      ISSUER: 'https://review-c.dev.account.gov.uk'
+      ISSUER: "https://review-c.dev.account.gov.uk"
+      CLIENTS: '[{"jwksEndpoint":"https://ipvstub.review-c.dev.account.gov.uk/.well-known/jwks.json","clientId":"5C584572","redirectUri":"https://erveje5km8.execute-api.eu-west-2.amazonaws.com/dev/redirect"}]'
       MESSAGERETENTIONPERIODDAYS: 345600 # Default: 4 days
+      AUTHSESSIONTTL: "604800" # 7 days in seconds
     build:
-      ISSUER: 'https://review-c.build.account.gov.uk'
+      ISSUER: "https://review-c.build.account.gov.uk"
+      CLIENTS: '[{"jwksEndpoint":"https://ipvstub.review-c.build.account.gov.uk/.well-known/jwks.json","clientId":"BD7B2A5D","redirectUri":"https://chspgdcmce.execute-api.eu-west-2.amazonaws.com/dev/redirect"}]'
       MESSAGERETENTIONPERIODDAYS: 345600 # Default: 4 days
+      AUTHSESSIONTTL: "604800" # 7 days in seconds
     staging:
-      ISSUER: 'https://review-c.staging.account.gov.uk'
+      ISSUER: "https://review-c.staging.account.gov.uk"
+      CLIENTS: '[{"jwksEndpoint":"https://chspgdcmce.execute-api.eu-west-2.amazonaws.com/dev/.well-known/jwks.json","clientId":"5C584572","redirectUri":"https://chspgdcmce.execute-api.eu-west-2.amazonaws.com/dev/redirect"}]'
```

Comment:
the well known endpoints in here need to be the real IPV Core urls, please check the slack thread [here](https://gds.slack.com/archives/C0440GZHUUX/p1677760990723289?thread_ts=1677752829.023559&cid=C0440GZHUUX)